### DISCUSSION TITLE
Is the Windows binary safe to use?

### DISCUSSION THREAD

**Discussion by MobinAskari** [minio/minio#20202] [Q&A] [answered]
Hello, I hope everyone's having a great day.
I downloaded the Windows binary from this URL: https://dl.min.io/server/minio/release/windows-amd64/minio.exe and upon trying to open it up, I'm met with a security warning from Windows:

![image](https://github.com/user-attachments/assets/67d8e832-f83d-4acb-8b63-72750dda1055)

If it matters, I'm using Windows 11 V23H2

**Comment by harshavardhana** [comment, marked as answer]
Our binaries are built via go - beyond that we do not do any real testing on windows. 

For production deployments we always recommend Linux

**Reply by ravindk89** [reply]
To add to this - to get rid of that notice, we would have to take extra steps to sign and coordinate verification with Microsoft/Windows.

As we do not intend for Windows builds to be used in any production environments, this is a low-to-no priority line item. 

You can always verify the binary against our available checksums at dl.min.io if you are unusure of the provenance of the binary.
